views/js/pciCreator/dev/mathEntryInteraction/creator/tpl/propertiesForm.tpl

```diff
@@ -11,6 +11,17 @@
 </div>
 
 <hr />
+<h3>{{__ "Special Settings"}}</h3>
+
+<div>
+    <label class="panel">
+        <input name="spaceBehavesLikeTab" type="checkbox" {{#if spaceBehavesLikeTab}}checked="checked"{{/if}}/>
+        <span class="icon-checkbox"></span>
+        {{__ "prevent white space"}}
```

Comment:
Tom once made a very useful comment about options and UX, that we should always tick a checkbox to _activate_ or _enable_ something, not the other way around which confuses the poor user's brain ;-)